frontend/src/components/noticeOfWork/applications/NoticeOfWorkApplication.js

```diff
@@ -25,24 +32,42 @@ const propTypes = {
   fetchNoticeOfWorkApplication: PropTypes.func.isRequired,
   noticeOfWork: CustomPropTypes.nowApplication.isRequired,
   createNoticeOfWorkApplication: PropTypes.func.isRequired,
-  location: PropTypes.shape({ search: PropTypes.string }).isRequired,
+  fetchMineRecordById: PropTypes.func.isRequired,
+  fetchImportedNoticeOfWorkApplication: PropTypes.func.isRequired,
+  history: PropTypes.shape({ push: PropTypes.func }).isRequired,
+  match: PropTypes.shape({
+    params: {
+      id: PropTypes.string,
+    },
+  }).isRequired,
+  // the following prop will be used in the future
+  // eslint-disable-next-line
+  formValues: CustomPropTypes.nowApplication.isRequired,
+  mines: PropTypes.arrayOf(CustomPropTypes.mine).isRequired,
 };
 
 export class NoticeOfWorkApplication extends Component {
   state = {
     currentStep: 0,
     isLoaded: false,
     associatedMineGuid: "",
+    isViewMode: true,
+    fixedTop: false,
   };
 
   componentDidMount() {
-    const params = queryString.parse(this.props.location.search);
-    const { id } = params;
+    window.scrollTo(0, 0);
```

Comment:
I'm not _sure_ this works as expected in all browsers. I have a vague memory of some complexity when I looked into this issue in the past, although it may not longer be true.

Reply:
I think you are correct, I will remove it from here (in the follow up PR) and we can address it in MDS-2449 (add scroll on every route change)